mod_reforged/hooks/config/strings.nut

```diff
@@ -1318,9 +1318,9 @@ foreach (vanillaDesc in vanillaDescriptions)
 		Fluff = "Burn them all!",
 		Requirement = "Bow",
 		Effects = [{
-			Type = ::UPD.EffectType.Passive,
+			Type = ::UPD.EffectType.Active,
 			Description = [
-				"A successful [Aimed Shot|Skill+aimed_shot] will now light the target tile on fire for 2 rounds and trigger an additional morale check for the target and adjacent enemies."
+				"Unlocks the [Flaming Arrow|Skill+rf_flaming_arrows_skill] skill with bows."
```

Comment:
no need to mention bows here when they are already mentioned in the requirement for this whole perk. So it should be applied that this perks effect (adding a new skill) only works while a bow is equipped.